target="_blank" uses need more attention

In properly hooking up eslinter to the sinopia editor, we have a few errors flagged re: target="_blank"

```
sinopia_editor/src/components/CanvasMenu.jsx
  52:79  error    Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener                                                                                                                                                                                                                                      react/jsx-no-target-blank
  53:57  error    Using target="_blank" without rel="noopener noreferrer" is a security risk: ...
  54:50  error    Using target="_blank" without rel="noopener noreferrer" is a security risk: ...
  55:55  error    Using target="_blank" without rel="noopener noreferrer" is a security risk: ...
  56:47  error    Using target="_blank" without rel="noopener noreferrer" is a security risk: ...
  57:46  error    Using target="_blank" without rel="noopener noreferrer" is a security risk: ...
  58:66  error    Using target="_blank" without rel="noopener noreferrer" is a security risk: ...
  59:50  error    Using target="_blank" without rel="noopener noreferrer" is a security risk: ...                                                                                                                                                             
  60:73  error    Using target="_blank" without rel="noopener noreferrer" is a security risk: ...                                                                                                                                                             
  61:69  error    Using target="_blank" without rel="noopener noreferrer" is a security risk: ...
  62:89  error    Using target="_blank" without rel="noopener noreferrer" is a security risk: ...
  63:43  error    Using target="_blank" without rel="noopener noreferrer" is a security risk: ...
  64:74  error    Using target="_blank" without rel="noopener noreferrer" is a security risk: ...

/Users/ndushay/ld4p/sinopia_editor/src/components/Footer.jsx
   8:106  error  Using target="_blank" without rel="noopener noreferrer" is a security risk: ...
  15:75   error  Using target="_blank" without rel="noopener noreferrer" is a security risk: ...
  16:177  error  Using target="_blank" without rel="noopener noreferrer" is a security risk: ...

/Users/ndushay/ld4p/sinopia_editor/src/components/NewsItem.jsx
  9:179  error  Using target="_blank" without rel="noopener noreferrer" is a security risk: ...

--- a/.eslintrc.js
+++ b/.eslintrc.js
@@ -44,7 +44,6 @@ module.exports = {
     "jsx-a11y/label-has-for": "warn", // see #173
     "no-console": "warn",
     "no-extra-semi": "off", // because it isn't that important
-    "react/jsx-no-target-blank": "warn", // see #174
     "react/prop-types": "warn" // see #175
   },
   overrides: [

--- a/__tests__/components/CanvasMenu.test.js
+++ b/__tests__/components/CanvasMenu.test.js
@@ -17,4 +17,10 @@ describe('<CanvasMenu />', () => {
   it('renders the FontAwesome closes icon', () =>{
     expect(wrapper.find(".close-icon")).toBeDefined()
   })
+
+  it('checks if links with target="_blank" also have rel="noopener noreferrer"', () => {
+    wrapper.find('a[target="_blank"]').forEach((node) => {
+      expect(node.prop('rel')).toEqual("noopener noreferrer")
+    })
+  })
 })

--- a/__tests__/components/Footer.test.js
+++ b/__tests__/components/Footer.test.js
@@ -12,4 +12,9 @@ describe('<Footer />', () => {
   it ('contains the ld4p link', () => {
     expect(wrapper.find('a[href="http://www.ld4p.org"]')).toBeDefined()
   })
+  it('checks if links with target="_blank" also have "noopener noreferrer" in rel attribute', () => {
+    wrapper.find('a[target="_blank"]').forEach((node) => {
+      expect(node.prop('rel')).toMatch(/noopener noreferrer/)
+    })
+  })
 })

--- a/__tests__/components/NewsItem.test.js
+++ b/__tests__/components/NewsItem.test.js
@@ -5,8 +5,15 @@ import { shallow, render } from 'enzyme'
 import NewsItem from '../../src/components/NewsItem'
 
 describe('<NewsItem />', () => {
+  const wrapper = shallow(<NewsItem />)
+
   it ('renders the latest news', () => {
-    const wrapper = shallow(<NewsItem />)
     expect(wrapper.find("div.news-item")).toBeDefined()
+  })
+
+  it('checks if links with target="_blank" also have rel="noopener noreferrer"', () => {
+    wrapper.find('a[target="_blank"]').forEach((node) => {
+      expect(node.prop('rel')).toEqual("noopener noreferrer")
+    })
   })
 })

--- a/package.json
+++ b/package.json
@@ -68,7 +68,7 @@
     "dev-build": "webpack --progress --mode development",
     "dev-build-test": "npm run dev-build && npm run test",
     "dev-start": "webpack-dev-server --config ./webpack.config.js --mode development",
-    "eslint": "eslint --max-warnings 100 --color -c .eslintrc.js --ext js,jsx ./src",
+    "eslint": "eslint --max-warnings 60 --color -c .eslintrc.js --ext js,jsx ./src",
     "jest-cov": "jest --coverage --colors",
     "jest-ci": "jest --ci --runInBand --coverage --reporters=default --reporters=jest-junit --colors  && cat ./coverage/lcov.info | coveralls",
     "test": "jest --colors"

--- a/src/components/CanvasMenu.jsx
+++ b/src/components/CanvasMenu.jsx
@@ -19,49 +19,49 @@ class CanvasMenu extends Component {
           <br></br>
           <br></br>
           <li>
-            <FontAwesomeIcon className="arrow-icon" icon={faArrowAltCircleRight} /> <a href="#" target="_blank"><strong>Sinopia User Dashboard</strong></a>
+            <FontAwesomeIcon className="arrow-icon" icon={faArrowAltCircleRight} /> <a href="#" target="_blank" rel="noopener noreferrer" ><strong>Sinopia User Dashboard</strong></a>
           </li>
           <br></br>
           <br></br>
           <li>
             <FontAwesomeIcon className="arrow-icon" icon={faArrowAltCircleRight} /> <a href="#"><strong>Contact Us</strong></a>
           </li>
-          <li><a href="#" target="_blank" className="menu-item">E-mail Sinopia group</a></li>
+          <li><a href="#" target="_blank" rel="noopener noreferrer" className="menu-item">E-mail Sinopia group</a></li>
           <li><a href="https://ld4.slack.com/messages/#sinopia" className="menu-item">Join Slack Channel</a></li>
           <br></br>
           <br></br>
           <li>
             <FontAwesomeIcon className="arrow-icon" icon={faArrowAltCircleRight} /> <a href="#"><strong>Training Resources</strong></a>
           </li>
-          <li><a href="#" target="_blank" className="menu-item">Intro to Profile Editor</a></li>
-          <li><a href="#" target="_blank" className="menu-item">Intro to Bibliographic Editor</a></li>
-          <li><a href="#" target="_blank" className="menu-item">Troubleshooting</a></li>
+          <li><a href="#" target="_blank" rel="noopener noreferrer" className="menu-item">Intro to Profile Editor</a></li>
+          <li><a href="#" target="_blank" rel="noopener noreferrer" className="menu-item">Intro to Bibliographic Editor</a></li>
+          <li><a href="#" target="_blank" rel="noopener noreferrer" className="menu-item">Troubleshooting</a></li>
           <br></br>
           <br></br>
           <li>
             <FontAwesomeIcon className="arrow-icon" icon={faArrowAltCircleRight} /> <a href="#"><strong>Website Usage</strong></a>
           </li>
-          <li><a href="#" target="_blank" className="menu-item">Policies</a></li>
-          <li><a href="#" target="_blank" className="menu-item">Terms of Use</a></li>
-          <li><a href="#" target="_blank" className="menu-item">Request Account</a></li>
+          <li><a href="#" target="_blank" rel="noopener noreferrer" className="menu-item">Policies</a></li>
+          <li><a href="#" target="_blank" rel="noopener noreferrer" className="menu-item">Terms of Use</a></li>
+          <li><a href="#" target="_blank" rel="noopener noreferrer" className="menu-item">Request Account</a></li>
           <br></br>
           <br></br>
           <li>
             <FontAwesomeIcon className="arrow-icon" icon={faArrowAltCircleRight} /> <a href="#"><strong>External Resources</strong></a>
           </li>
-          <li><a href="http://www.getty.edu/research/tools/vocabularies/aat/" target="_blank" className="menu-item">Art & Architecture Thesaurus</a></li>
-          <li><a href="https://www.discogs.com/search/" target="_blank" className="menu-item">Discogs</a></li>
-          <li><a href="http://www.geonames.org/" target="_blank" className="menu-item">Geonames</a></li>
-          <li><a href="https://translate.google.com/" target="_blank" className="menu-item">Google Translate</a></li>
-          <li><a href="https://www.imdb.com/" target="_blank" className="menu-item">IMDB</a></li>
-          <li><a href="http://www.isni.org/" target="_blank" className="menu-item">ISNI</a></li>
-          <li><a href="http://id.loc.gov/authorities/names.html" target="_blank" className="menu-item">Library of Congress</a></li>
-          <li><a href="https://musicbrainz.org/" target="_blank" className="menu-item">MusicBrainz</a></li>
-          <li><a href="http://www.share-vde.org/sharevde/clusters?l=en" target="_blank" className="menu-item">Share-VDE</a></li>
-          <li><a href="http://www.loc.gov/pictures/collection/tgm/" target="_blank" className="menu-item">Thesaurus of Graphic Materials</a></li>
-          <li><a href="http://www.getty.edu/research/tools/vocabularies/tgn/index.html" target="_blank" className="menu-item">Thesaurus of Geographical Names</a></li>
-          <li><a href="https://viaf.org/" target="_blank" className="menu-item">VIAF</a></li>
-          <li><a href="https://www.wikidata.org/wiki/Wikidata:Main_Page" target="_blank" className="menu-item">Wikidata</a></li>
+          <li><a href="http://www.getty.edu/research/tools/vocabularies/aat/" target="_blank" rel="noopener noreferrer" className="menu-item">Art & Architecture Thesaurus</a></li>
+          <li><a href="https://www.discogs.com/search/" target="_blank" rel="noopener noreferrer" className="menu-item">Discogs</a></li>
+          <li><a href="http://www.geonames.org/" target="_blank" rel="noopener noreferrer" className="menu-item">Geonames</a></li>
+          <li><a href="https://translate.google.com/" target="_blank" rel="noopener noreferrer" className="menu-item">Google Translate</a></li>
+          <li><a href="https://www.imdb.com/" target="_blank" rel="noopener noreferrer" className="menu-item">IMDB</a></li>
+          <li><a href="http://www.isni.org/" target="_blank" rel="noopener noreferrer" className="menu-item">ISNI</a></li>
+          <li><a href="http://id.loc.gov/authorities/names.html" rel="noopener noreferrer" target="_blank" className="menu-item">Library of Congress</a></li>
+          <li><a href="https://musicbrainz.org/" target="_blank" rel="noopener noreferrer" className="menu-item">MusicBrainz</a></li>
+          <li><a href="http://www.share-vde.org/sharevde/clusters?l=en" target="_blank" rel="noopener noreferrer" className="menu-item">Share-VDE</a></li>
+          <li><a href="http://www.loc.gov/pictures/collection/tgm/" target="_blank" rel="noopener noreferrer" className="menu-item">Thesaurus of Graphic Materials</a></li>
+          <li><a href="http://www.getty.edu/research/tools/vocabularies/tgn/index.html" rel="noopener noreferrer"  target="_blank" className="menu-item">Thesaurus of Geographical Names</a></li>
+          <li><a href="https://viaf.org/" target="_blank" rel="noopener noreferrer" className="menu-item">VIAF</a></li>
+          <li><a href="https://www.wikidata.org/wiki/Wikidata:Main_Page" target="_blank" rel="noopener noreferrer" className="menu-item">Wikidata</a></li>
         </ul>
       </div>
     );

--- a/src/components/Footer.jsx
+++ b/src/components/Footer.jsx
@@ -5,15 +5,15 @@ import React from 'react'
 const Footer = () => (
   <div id="footer" className="row">
     <div id="footer-image" className="col-xs-6 col-sm-1">
-      <a className='footer-image' rel="license" href="http://creativecommons.org/publicdomain/zero/1.0/" target="_blank">
+      <a className='footer-image' rel="license noopener noreferrer" href="http://creativecommons.org/publicdomain/zero/1.0/" target="_blank">
         <img alt="CC0" src="http://i.creativecommons.org/p/zero/1.0/88x31.png" />
       </a>
     </div>
     <div id="footer-text" className="col-xs-6 col-sm-10">
       <p>
         <small>
-        Sinopia is a project of <a rel="grant" href="http://www.ld4p.org" target="_blank"> Linked Data for Production 2 (LD4P2)</a>, generously funded by the Andrew W. Mellon Foundation.
-        All metadata available on Sinopia are published free of restrictions, under the terms of the <a rel="license" href="https://creativecommons.org/publicdomain/zero/1.0/" target="_blank">Creative Commons CC0 1.0 Universal Public Domain Dedication</a>.
+        Sinopia is a project of <a rel="grant noopener noreferrer" href="http://www.ld4p.org" target="_blank"> Linked Data for Production 2 (LD4P2)</a>, generously funded by the Andrew W. Mellon Foundation.
+        All metadata available on Sinopia are published free of restrictions, under the terms of the <a rel="license noopener noreferrer" href="https://creativecommons.org/publicdomain/zero/1.0/" target="_blank">Creative Commons CC0 1.0 Universal Public Domain Dedication</a>.
         </small>
       </p>
     </div>

--- a/src/components/NewsItem.jsx
+++ b/src/components/NewsItem.jsx
@@ -6,7 +6,7 @@ const NewsItem = () => (
   <div className="news-item">
     <h1> Latest News </h1>
     <ul>
-      <li>We are looking for a Wikipedian in residence to join our team and help identify how libraries can contribute and relate their information with that of the open Web. <a target="_blank" href="http://m.rfer.us/STANFORDlAy3UB">Learn more</a></li>
+      <li>We are looking for a Wikipedian in residence to join our team and help identify how libraries can contribute and relate their information with that of the open Web. <a target="_blank" rel="noopener noreferrer" href="http://m.rfer.us/STANFORDlAy3UB">Learn more</a></li>
       <p></p>
       <li>We achieved our first milestone on November 1. The Profile Editor has been released into production, and we are actively testing it.</li>
     </ul>

